FTHEL8/Menu/AddMenu.cs

```diff
@@ -0,0 +1,319 @@
+                foreach (Employee employee in employees)
+                {
+                    Console.Write(employee.EmployeeId + " ");
+                }
+                Console.WriteLine();
+
+                Console.Write("Enter Project Leader from the list: ");
+                string projectLeaderId = Console.ReadLine() ?? "";
+                List<Class> classList = Database.ReadClassesAsync().Result;
+                foreach (Class class_ in classList)
+                {
+                    Console.Write(class_.Name + " ");
+                }
+                Console.WriteLine();
+
+                Console.Write("Enter Class Name from the list: ");
+                string className = Console.ReadLine() ?? "";
+
+                bool validProjectLeader = employees.Any(e => e.EmployeeId == projectLeaderId);
+                bool validClass = classList.Any(c => c.Name == className);
+
+                if (validProjectLeader && validClass)
+                {
+                    bool success = Database.AddProjectAsync(projectName, description, deadline, projectLeaderId, className).Result;
+
+                    if (success)
+                    {
+                        Console.WriteLine("Project added successfully!");
+                    }
+                    else
+                    {
+                        Console.WriteLine("Failed to add project. Please check your input.");
+                    }
+                }
+                else
+                {
+                    Console.WriteLine("Invalid Project Leader or Class Name. Please enter valid values.");
+                }
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine(ex.Message);
+            }
+        }
+
+        private void AddEmployee()
+        {
+            try
+            {
+                Console.Write("Enter Employee ID: ");
+                string employeeId = Console.ReadLine() ?? "";
+                bool isNotValidEmployeeId = Database.ReadEmployeesAsync().Result.Any(e => e.Name == employeeId);
+                if (isNotValidEmployeeId && string.IsNullOrWhiteSpace(employeeId))
+                {
+                    Console.WriteLine("This employee ID is already in the database or can't be added!");
+                    return;
+                }
+
+                Console.Write("Enter Name: ");
+                string name = Console.ReadLine() ?? "";
+
+                Console.Write("Enter Phone Number: ");
+                string phoneNumber = Console.ReadLine() ?? "";
+
+                Console.Write("Enter Email: ");
+                string email = Console.ReadLine() ?? "";
+
+                Console.Write("Enter Position: ");
+                string position = Console.ReadLine() ?? "";
+
+                Console.Write("Enter Salary, must be a number: ");
+                int salary = int.Parse(Console.ReadLine() ?? "");
```

Comment:
Attól, hogy must a user még lehet buta vagy figyelmetlen. Az int.Parse pedig el fog szálni mint a győzelmi zászló az az asdf inputra, de ugyan úgy az 1215,26-ra is, pedig az utóbbi number.